src/viur/core/modules/user.py

```diff
@@ -800,7 +800,7 @@ def otp(self, *args, **kwargs):
 
         # Check if the OTP device has a time drift
 
-        timedriftchange = float(res) - otp_user_conf["timedrift"]
+        timedriftchange = float(res) - otp_user_conf.timedrift
```

Comment:
This is still broken!
```suggestion
        timedriftchange = float(res) - otp_user_conf["timedrift"]
```